ward/terminal.py

```diff
@@ -370,7 +487,45 @@ def output_all_test_results(
         output_tests = self.runtime_output_strategies.get(
             self.test_output_style, output_test_per_line
         )
-        all_results = output_tests(fail_limit, test_results_gen)
+
+        spinner = SpinnerColumn(style="pass.textonly")
+        bar = BarColumn(complete_style="pass.textonly")
+        progress = Progress(
+            spinner,
+            TimeElapsedColumn(),
+            bar,
+            "[progress.percentage]{task.percentage:>3.0f}%",
+            "[progress.percentage][{task.completed} / {task.total}]",
+            TimeRemainingColumn(),
```

Comment:
I'm not sure about the value of the time remaining column. I actually wasn't sure what the number represented when viewing the output. I think we either get rid of it or make it clearer what it represents. In my own experience I don't think the time remaining in a test suite is something predictable without looking at historical runs.

Reply:
Makes a ton of sense; I'll remove it.